skpro/regression/linear/_glm.py

```diff
@@ -18,11 +21,34 @@ class GLMRegressor(BaseProbaRegressor):
 
     Parameters
     ----------
+    family : str, default : "Normal"
+        Available options are
+        1.Normal
+        2.Poisson
+        3.Gamma
+    link : str
+        Available safe options are
+        Normal : Log, Identity, InversePower
+        Poisson : Log, Identity, Sqrt
+        Gamma : Log, Identity, InversePower
+    offset_var : pd.Index([str]) or int, default = None
```

Comment:
why is this a pd.index? I was thinking index element, there is no benefit in passing this wrapped in an index